com.unity.render-pipelines.high-definition/Editor/VFXGraph/Utility/VFXHDRPSettingsUtility.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace UnityEditor.VFX.HDRP
 {
-    public static class VFXHDRPSettingsUtility
+    static class VFXHDRPSettingsUtility
     {
         public static void RefreshVfxErrorsIfNeeded(ref bool needRefreshVfxErrors)
```

Comment:
Do we also need to set RefreshVfxErrorsIfNeeded() to internal or changing the parent class in enough ?

Reply:
No, otherwise, this function won't be accessible by any user code, we have :
```internal class VFXHDRPSettingsUtility
{
 public //reachable in HDRP
}
```